barman/backup.py

```diff
@@ -134,6 +134,21 @@ def get_available_backups(self, status_filter=DEFAULT_STATUS_FILTER):
                 backups[key] = value
         return backups
 
+    def get_available_full_backups(self, status_filter=DEFAULT_STATUS_FILTER):
```

Comment:
not sure about this. Have you thought about implementing this as an option for the `get_available_backups` method? something like:
```python
get_available_backups(self, status_filter=DEFAULT_STATUS_FILTER, incrementals=false)
```

Also please don't assume that the above is correct by default, search for possible implications in the code if you haven't already.

Reply:
just to be clear... my general sensation is that we are sorta duplicating code just adding `_full` in the name methods... duplicating is not even the right word here... is more... overloading? is clear what I mean?

Reply:
Right, I've been taking a look and a I have a similar sensation. Maybe we ware able to modify the existing methods they look less like copy'n'paste.

Reply:
Yeah, i thought about modifying the existing get_avaialable_backups! I just thought about creating this other one so i would not change the signature of the other one and because this existing method is used in several places and i would end up having to change them all! But i agree about being a little repetitive here! Also this new method uses the existing one. If you guys agree, i can modify the existing method and remove this new one! What you guys think?